test/EurekaClient.test.js

```diff
@@ -112,25 +112,41 @@ describe('Eureka client', () => {
     let client;
     let registerSpy;
     let fetchRegistrySpy;
+    let heartbeatsSpy;
+    let registryFetchSpy;
     before(() => {
       config = makeConfig();
       client = new Eureka(config);
-      registerSpy = sinon.stub(client, 'register').callsArg(0);
-      fetchRegistrySpy = sinon.stub(client, 'fetchRegistry').callsArg(0);
     });
 
     after(() => {
       registerSpy.restore();
       fetchRegistrySpy.restore();
+      heartbeatsSpy.restore();
+      registryFetchSpy.restore();
     });
 
     it('should call register and fetch registry', (done) => {
+      registerSpy = sinon.stub(client, 'register').callsArg(0);
+      fetchRegistrySpy = sinon.stub(client, 'fetchRegistry').callsArg(0);
+
```

Comment:
Build is failing because heartbeats isn't stubbed in this test so it's trying to make actual requests and getting connection refused.